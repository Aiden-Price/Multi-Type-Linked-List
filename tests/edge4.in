NEW 1
newNode
NEW 1
{List 0}
NEW 1
{List 1}
INSERT 0 0 {List 2}