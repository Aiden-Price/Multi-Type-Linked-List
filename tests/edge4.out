List 0: newNode
List 1: {List 0}
List 2: {List 1}
